test/e2e/thv-operator/virtualmcp/virtualmcp_discovered_mode_test.go

```diff
@@ -337,13 +337,18 @@ var _ = Describe("VirtualMCPServer Discovered Mode", Ordered, func() {
 			if targetToolName != "" {
 				GinkgoWriter.Printf("Testing tool call for: %s\n", targetToolName)
 
+				// Use a longer timeout for the tool call since it involves fetching an external URL
+				toolCallCtx, toolCallCancel := context.WithTimeout(context.Background(), 60*time.Second)
+				defer toolCallCancel()
+
 				callRequest := mcp.CallToolRequest{}
 				callRequest.Params.Name = targetToolName
 				callRequest.Params.Arguments = map[string]any{
-					"url": "https://example.com",
+					// Use httpbin.org which is fast and reliable for testing
+					"url": "https://httpbin.org/get",
```

Comment:
if this is giving us issues we could also spawn a localhost server maybe? Or just use an IP address